Instructions/08-perform-data-analysis-in-power-bi-desktop.md

```diff
@@ -59,41 +59,37 @@ In this task, you'll set up the environment for the lab by signing in to Power B
 
 In this task, you'll set up the environment for the lab by creating a dataset.
 
-*Important: If you've already published the dataset in the **Create a Power BI Dashboard** lab, continue from the next task.*
+*Important: If you've already published the dataset, please move to the next task.*
 
```

Comment:
This assumes that the student knows and completed one of the previous labs. I would just use
 _If you've already published the dataset in a previous lab, please move to the next task._